src/main/java/com/nakal/utils/YamlReader.java

```diff
@@ -0,0 +1,46 @@
+package com.nakal.utils;
+
+import org.yaml.snakeyaml.Yaml;
```

Comment:
where did we get this from? I don't any dependencies been added here. Is it internal dep of something else.

Reply:
snakeyaml dependency is already part of project

Reply:
oh cool

Reply:
Can this be merged?